bpf/lib/lb.h

```diff
@@ -935,7 +935,7 @@ static __always_inline int lb6_local(const void *map, struct __ctx_buff *ctx,
 		/* See lb4_local comment */
 		if (state->rev_nat_index == 0) {
 			state->rev_nat_index = svc->rev_nat_index;
-			ct_update6_rev_nat_index(map, tuple, state);
```

Comment:
FYI: even if we don't rely on the compiler anymore to check the types, we could easily rely on Coccinelle instead to check that the second parameter is one of the two expected types.